cmd/add.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/railwayapp/cli/entity"
+	"github.com/railwayapp/cli/ui"
+)
+
+func (h *Handler) Add(ctx context.Context, req *entity.CommandRequest) error {
+	projectCfg, err := h.cfg.GetProjectConfigs()
+	if err != nil {
+		return err
+	}
+
+	project, err := h.ctrl.GetProject(ctx, projectCfg.Project)
+	if err != nil {
+		return err
+	}
+
+	pluginRequest := strings.TrimSpace(req.Args[0])
+	allowCreation, pluginsAvailable, err := h.ctrl.PluginExists(ctx, pluginRequest, project.Id)
```

Comment:
Triple return is almost never the move. Additionally, the plugin request should take the projectID as part of the payload. Make an entity

Suggested abstraction: 
```
[]Plugin, err := h.ctrl.GetAvailablePlugins(ctx, project.Id)
```